modules/tinymce/src/core/main/ts/api/Resource.ts

```diff
@@ -10,6 +10,7 @@ import ScriptLoader from './dom/ScriptLoader';
 interface Resource {
   load: <T = any>(id: string, url: string) => Promise<T>;
   add: (id: string, data: any) => void;
+  unload: (id: string) => void;
```

Comment:
I was unsure if we should add a changelog item or not since it's mostly an API we need to testing not really something a customer would need also there is no api docs for any of the resource API so it is kind of hidden already. But I guess they too could be trying to load the editor multiple times with the same resource id. I added a changelog item.